src/windows/WATNav.js

```diff
@@ -46,5 +69,37 @@ navigateBack = function (e) {
   return true;
 }
 
+createBackButton = function () {
+    var wrapperDiv = document.createElement("div");
+
+    wrapperDiv.id = "backbutton-wrapper";
+    wrapperDiv.style.zIndex = WAT.components.webView.style.zIndex + 100;
+    wrapperDiv.style.position = "absolute";
+    wrapperDiv.style.top = "0";
+    wrapperDiv.style.left = "0";
+    wrapperDiv.style.padding = "50px";
+    wrapperDiv.style.width = "150px";
+    wrapperDiv.style.height = "50px";
+
+    var btn = document.createElement("btn");
+    btn.classList.add("win-backbutton");
```

Comment:
Why are you adding the win-backbutton class?

In the Universal WAT, this class includes the following styles:

### wrapper-styles.css

````
.win-backbutton {
    background-color: transparent;
    border-color: rgba(0,0,0,0.7);
    color: rgb(0, 0, 0);
    opacity: 0.6;
}

    .win-backbutton:hover {
        background-color: rgba(0,0,0,0.2);
        opacity: 1;
    }
`````

### base.css

````
/*
Back buttons.
*/

.disabled {
    opacity: 0.3;
}

.win-backbutton, .win-back {
    display: inline-block;
    min-width: 0;
    min-height: 0;
    background-clip: border-box;
    box-sizing: border-box;
    border-radius: 50%;
    border-width: 2px;
    border-style: solid;
    padding: 0;
    text-align: center;

    /* Normal sizing. */
    width: 41px;
    height: 41px;
    font-size: 14pt;
    line-height: 37px; /* line-height must match the content box height. */
    vertical-align: baseline;
}

.win-backbutton:hover, .win-backbutton:hover:active {
    border-width: 2px;
    border-style: solid;
    border-radius: 50%;
}
.win-backbutton::before, .win-back::before {
    font-family: "Segoe UI Symbol";
    font-weight: normal;
    content: "\E0D5";
    vertical-align: 50%;
}
.win-backbutton:-ms-lang(ar, dv, fa, he, ku-Arab, pa-Arab, prs, ps, sd-Arab, syr, ug, ur, qps-plocm)::before, .win-back:-ms-lang(ar, dv, fa, he, ku-Arab, pa-Arab, prs, ps, sd-Arab, syr, ug, ur, qps-plocm)::before {
    content: "\E0AE";
}
.win-backbutton:disabled, .win-backbutton:disabled:active,
.win-navigation-backbutton:disabled, .win-navigation-backbutton:disabled:active {
    visibility: hidden;
}

button.win-navigation-backbutton, button.win-navigation-backbutton:active, 
button.win-navigation-backbutton:hover, button.win-navigation-backbutton:active:hover
{
    display: inline-block;
    padding: 2px;
    min-height: 0px;
    min-width: 0px;
    
    /* Overwrite default background and border styles from HTML button element */
    background-color: transparent;
    border: none;

    /* Normal Sizing */
    height: 45px;
    width: 45px;
}
`````

### Windows Phone's default.css

````
.win-settingsflyout .win-backbutton {
    visibility: collapse;
    display: none;
}
````